jest.setup.js

```diff
@@ -0,0 +1,39 @@
+import '@testing-library/jest-dom';
+import path from 'path';
+import glob from 'glob';
+import forEach from 'lodash/forEach';
+import join from 'lodash/join';
+import uniq from 'lodash/uniq';
+import upperFirst from 'lodash/upperFirst';
+import Vue from 'vue';
+
+/* Globally register all the availbale components in the base & core directories
+ * so that they can be found by jest without any complications.
+ *
+ * Solution is based on the following example:
+ * https://github.com/nuxt/components/issues/58#issuecomment-709961688
+ */
+const globalComponentPaths = [
+  './components/**/base/**/*.vue',
+  './components/**/core/**/*.vue',
+];
+forEach(globalComponentPaths, (globalComponentPath) => {
```

Comment:
I understand using lodash for join and uniq, but forEach is native, so we should use that instead :)